spec/datadog/tracing/integration_spec.rb

```diff
@@ -577,6 +599,81 @@ def sample!(trace)
         expect(stats[:transport].internal_error).to eq(0)
       end
     end
+
+    context 'with agent rates' do
+      before do
+        WebMock.enable!
+        stub_request(:post, %r{/v0.4/traces}).to_return(status: 200, body: service_rates.to_json)
+      end
+
+      after { WebMock.disable! }
+
+      let(:service_rates) { { rate_by_service: { 'service:kept,env:' => 1.0, 'service:dropped,env:' => Float::MIN } } }
```

Comment:
This `Float::MIN` made me curious, and I did some grepping in the codebase. I see that we reference it in `rate_by_key_sampler_spec.rb` as being "for testing purposes".

My reading from that is that this value would never be returned by the agent. Is that really the case? If so, can I convince you to add a bit of context here about what this value means as well?

**Edit**: A second pass left me wondering: is `Float::MIN` relevant at all to what we're testing here? Could we instead remove it?